backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4471,3 +4471,23 @@ databaseChangeLog:
               - column:
                   name: latest_recorded_timestamp
                   descending: true
+  - changeSet:
+      id: add-queuename-column-report-stream-response
+      author: tsv7@cdc.gov
+      comment: Add column queue name to report stream response table to differentiate the origin of the exception
+      changes:
+        - tagDatabase:
+            tag: add-queuename-column-report-stream-response
+        - addColumn:
+            tableName: report_stream_response
+            columns:
+              - column:
+                  name: queue_name
+                  type: text
+                  remarks: Tracks the individual's preference for receiving test results
+      rollback:
```

Comment:
rollback is not required for addColumn https://docs.liquibase.com/change-types/add-column.html#:~:text=PostgreSQL,Supported